src/sonic-yang-models/tests/yang_model_tests/tests/aaa.json

```diff
@@ -0,0 +1,13 @@
+{
+    "AAA_TEST": {
+        "desc": "Configure an authentication type in AAA table."
+    },
+    "AAA_TEST_WRONG_TYPE": {
+        "desc": "Configure a wrong type in AAA table.",
+	"eStrKey": "InvalidValue"
```

Comment:
Seems to be alignment issues in some of these files due to a mixture of tabs and spaces. Please replace tabs with spaces everywhere.

Reply:
Addressed